### DISCUSSION TITLE
RTSP proxy for five camera's (newbee support)

### DISCUSSION THREAD

**Discussion by PerunNL** [bluenviron/mediamtx#1379] [Questions]
The stream of my camera's is often interrupted. The connection is lost or fully grey. All camera's are wired. I expect that the cameras can't handle the amount of requests of multiple devices. 

Therefore I'd like to make a proxy. I have Raspberry pi 3B. When I use MotionEyeOS the same happens, also with motion tracking and other services disabled. I try to install rstp-simple-server but I still have no picture from the server. The source urls are checked in vlc.

What is the best OS for my device? I tried raspbian-buster, raspios-bullseye and raspbian-stretch-lite. Always with rstp-simple-server 0.21.1. I read tutorials and the documentation and set proxied settings. If I do a portscan only ssh appears. Is there anything else I need to do? Do I need to set up ffmpeg somewehere? I'm lost

**Comment by PerunNL** [comment]
After reducing the settings file to 13 lines I have a vision! The picture disappears after a few seconds

Used version: rtsp-simple-server_v0.21.1_linux_arm64v8.tar.gz

**Comment by aler9** [comment]
Write down the IP address of you Raspberry Pi. You can obtain it by running:

```
hostname -I
```

Then connect to the server hosted on the Raspberry Pi:

```
rtsp://raspberry-pi-ip:8554/mycamera
```

In case of grey / dropped frames, it means that your LAN is not capable of holding it, therefore try decreasing the bitrate of the stream in the camera settings.

[a comment by an automated account is not shown]
